test/table.ts

```diff
@@ -1178,9 +1187,31 @@ describe('BigQuery/Table', () => {
       });
     });
 
+    // TODO this doesn't work without promises or callbacks, refactor as
```

Comment:
We're going to want to fix this before we land :)

Reply:
This is related to #640, i'll bundle it with this PR then.

Reply:
#640 is resolved via #647 and has been merged to master. this test is no longer valid and has been removed